doc/source/ray-dashboard.rst

```diff
@@ -0,0 +1,357 @@
+Actor Titles
+############
+Name of an actor and its arguments.
+
+State
+#####
+State of an actor. 
+
+- 0: Alive
+- 1: Reconstructing
+- 2: Dead
+
+Pending
+#######
+Number of pending tasks for this actor.
+
+Excuted
+#######
+Number of executed tasks for this actor.
+
+NumObjectIdsInScope
+###################
+Number of object IDs in scope for this actor. 
+In scope means this object IDs are not supposed to be evicted. For example, if the objectID is locally referenced, it is in scope because this objectID should not be garbage collected to ensure correctness of Ray applications.
+
+NumLocalObjects
+###############
+Number of objectIDs that are in this actor's local memory. Only big objects (>100KB) are residing in a plasma objects and other small objects are staying in the local memory.
+
+UsedLocalObjectMemory
+#####################
+Memory usage of local objects of this actor.
+
+kill actor
+##########
+Button to kill an actor in a cluster. It is corresponding to ``ray.kill``. 
+
+profile for
+###########
+Button to run profiling. We currently support 10s, 30s and 60s profiling. It requires passwordless ``sudo``.
+
+Infeasible Actor Creation
+#########################
+Red colored actors. It happens when an actor requires more resources than a Ray cluster can provide.
+
+Pending Actor Creation
+######################
+Yellow colored actors. It happens when there is no available resource for this actor because it is already taken by other tasks / actors.
+
+Actor Hierarchy
+###############
+The logical view renders actor information in a tree format. To illustrate this, in the code block below, the ``Parent`` actor creates two ``Child`` actors and each ``Child`` actor creates one ``GrandChild`` actor.
+This relationship is visible in the dashboard *Logical View* tab.
+
+.. code-block:: python
+
+  import ray
+  ray.init()
+
+  @ray.remote
+  class Grandchild:
+      def __init__(self):
+          pass
+
+  @ray.remote
+  class Child:
+      def __init__(self):
+          self.grandchild_handle = Grandchild.remote()
+  
+  @ray.remote
+  class Parent:
+      def __init__(self):
+          self.children_handles = [Child.remote() for _ in range(2)]
+
+  parent_handle = Parent.remote()
+
+You can see that the dashboard shows the parent/child relationship as expected. 
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/Logical-view-basic.png
+    :align: center
+
+Ray Config
+~~~~~~~~~~~~
+This is the configuration set in ``cluster.yaml`` for the autoscaler mode. Check `Cluster.yaml reference <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-full.yaml>`_ to learn it in more details.
+
+
+Tune (Experimental)
+~~~~~~~~~~~~~~~~~~~
+Trial ID
+########
+Trial IDs for hyperparameter tuning.
+
+Job ID
+######
+Job IDs for hyperparameter tuning.
+
+STATUS
+######
+Status of each trial.
+
+Start Time
+##########
+Start time of each trial.
+
+Hyperparameters
+###############
```

Comment:
Can you do *bold* instead of a new header? The new header takes up a lot of vertical space: 
```rst
**Trial ID**: Trial IDs for hyperparameter tuning.
**Job ID**: Job IDs for hyperparameter tuning.
**Status**: Status of each trial
**Start Time**: Start time of each trial.